README.md

```diff
@@ -13,18 +13,19 @@ projects.
-| SHARPNESS_THRESHOLD  | 60                                      | Threshold, in %, an image's sharpness must meet to pass quality check |
+| Environment variable   | Default                                 | Description                                                                                                                                    |
+|------------------------|-----------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| STORAGE_ENGINE         | file                                    | Either `s3` or `file`                                                                                                                          |
+| S3_BUCKET              | `null`                                  | Name of the S3 bucket to store files in when using `s3` engine                                                                                 |
+| ROOT_LOGGING_LEVEL     | `info`                                  | Root logging level                                                                                                                             |
+| ANTIVIRUS_ENABLED      | `false` in development `true` in docker | Whether AntiVirus is enabled                                                                                                                   |
+| ANTIVIRUS_HOST         | localhost                               | Network host of the antivirus service                                                                                                          |
+| ANTIVIRUS_PORT         | 3310                                    | Network port of the antivirus service                                                                                                          |
+| ANTIVIRUS_TIMEOUT      | 30000                                   | Virus scanning timeout                                                                                                                         |
+| CONTEXT_PATH           | /v1                                     | Context path for service                                                                                                                       |
+| ENABLE_QUALITY_CHECK   | false                                   | Whether to enable OCR quality check                                                                                                            |
+| SHARPNESS_THRESHOLD    | 60                                      | Threshold, in %, an image's sharpness must meet to pass quality check                                                                          |
+| SPRING_PROFILES_ACTIVE | `development`                           | Only needs to be set to "development" when running in docker with [localstack](https://github.com/localstack) for a mock local AWS environment |
```

Comment:
I would probably go for maybe adding another type to `STORAGE_ENGINE`, like s3_endpoint or just adding another env variable like `BUCKET_URL`, rather than adding a profile. Because you could be in development mode, but opt for local storage, rather than localstack